data/fin.lean

```diff
@@ -2,8 +2,19 @@ import data.nat.basic
 
 open fin nat
 
-/-- Embedding of `fin n` in `fin (n+1)` -/
-def raise_fin {n : ℕ} (k : fin n) : fin (n + 1) := ⟨val k, lt_succ_of_lt (is_lt k)⟩
+namespace fin
+
+variable {n : ℕ}
+
+def raise (k : fin n) : fin (n + 1) := ⟨val k, lt_succ_of_lt (is_lt k)⟩
```

Comment:
please add the comment `/-- Embedding of `fin n` in `fin (n+1)` -/` back.
This is a user comment, i.e. you see it a tooltip.